x-pack/plugins/alerting/server/rules_client/methods/disable.ts

```diff
@@ -76,6 +76,7 @@ async function disableWithOCC(context: RulesClientContext, { id }: { id: string
       updateMeta(context, {
         ...attributes,
         enabled: false,
+        running: false,
```

Comment:
**question:** It's possible the rule is still running when disabling, should we keep the value unchanged here?